ui/app/builds/detail/build-log/pncBuildDetailBuildLogPage.js

```diff
@@ -25,19 +25,19 @@
       sshCredentials: '<'
     },
     templateUrl: 'builds/detail/build-log/pnc-build-detail-build-log-page.html',
-    controller: [Controller]
+    controller: ['authService', Controller]
   });
 
 
-  function Controller() {
+  function Controller(authService) {
     const $ctrl = this;
 
     // -- Controller API --
     $ctrl.logFileName = null;
     $ctrl.sshCredentialsBtn = {
       clicked: false
     };
-
+    $ctrl.currentUser = authService.getPrinciple();
```

Comment:
I also noticed that `authService.isCurrentUser()` exists and could make your condition a little bit easier, but I will leave it up to you, both solutions are correct, thanks for raising this improvement!

https://github.com/project-ncl/pnc-web-ui-angularjs/blob/master/ui/app/common/authentication/authService.js#L47

Reply:
aha yes, this is the better way to validate the user. I didn't notice that. Updated and merged the code :)